### ISSUE TITLE
OpenSSL3.x - no QUIC support

### ISSUE THREAD

**Issue by Zajsoft** [eilandert/deb.myguard.nl#2]
Hello,
just now:
```
Get:1 http://deb.myguard.nl noble/main amd64 libssl3 amd64 3.0.14-10myguard1~noble [2235 kB]
Get:2 http://deb.myguard.nl noble/main amd64 openssl amd64 3.0.14-10myguard1~noble [1224 kB]

/usr/sbin/nginx: undefined symbol: SSL_set_quic_method, version OPENSSL_3.0.0

```
:_(

Hotfix - manual install(downgrade)
https://deb.myguard.nl/pool/main/o/openssl/openssl_3.0.13-10myguard2~jammy_amd64.deb
https://deb.myguard.nl/pool/main/o/openssl/libssl3_3.0.13-10myguard2~jammy_amd64.deb

**Comment by eilandert** [comment]
Hi thank you, seems the upstream has a branch for 3.0.14 without quic, I'm going to repair it now.

**Comment by eilandert** [comment]
Reverted back to 3.0.13+quic on the apt repo

**Comment by Zajsoft** [comment]
Hi, nope, thank to you for nice useful repo :-) And fast repair :-)

BTW packages are not tested before uploading to the repository, or what led to this situation?

**Comment by eilandert** [comment]
> Hi, nope, thanks to you for nice useful repo :-) And fast repair :-)
> 
> BTW packages are not tested before uploading to the repository, or what led to this situation?

Manual error, the upstream ( https://github.com/quictls/openssl/ ) has a new tag 3.0.14, which my buildsystem had already rejected. Due to errors in the past on that upstream I assumed they made an error in the release chain and I forced it anyway as I usually have to do (last time they released "Opernssl" for example, the release for that there was no release), which I shouldn't have done. 

My apologies, assumption is the mother of all f*ckups. I can only hope Quic comes to regular OpenSSL soon.

**Comment by eilandert** [comment]
Well, I couldn't resist and fixed a 3.0.14+quic anyway, sigh
